File: queue/src/main/java/br/com/queue/dtos/customer/statistics/ResponseCustomerDashBoardDto.java
package br.com.queue.dtos.customer.statistics;

import java.util.List;

public record ResponseCustomerDashBoardDto(
        ResponseCountTotalCustomersStatisticsDto countTotalCustomersStatistics,
        List<ResponseCustomersCreatedByMonthStatisticsDto> customersCreatedByMonthStatistics
) {
}

File: queue-frontend/src/app/dtos/customer/statistics/ResponseCustomerDashBoardDto.ts
import { ResponseCountTotalCustomersStatisticsDto } from "./ResponseCountTotalCustomersStatisticsDto";
import { ResponseCustomersCreatedByMonthStatisticsDto } from "./ResponseCustomersCreatedByMonthStatisticsDto";

export interface ResponseCustomerDashBoardDto{
  countTotalCustomersStatistics: ResponseCountTotalCustomersStatisticsDto;
  customersCreatedByMonthStatistics: ResponseCustomersCreatedByMonthStatisticsDto[];
}
